.github/workflows/ci.yml

```diff
@@ -97,6 +98,7 @@ jobs:
         uses: burrunan/gradle-cache-action@v1.10
         with:
           job-id: latestDepTest
```

Comment:
It would be nice to try to save just this one to another key but can try it later.